readthedocs/settings/base.py

```diff
@@ -718,7 +718,8 @@ def DOCKER_LIMITS(self):
 
     # Organization settings
     RTD_ALLOW_ORGANIZATIONS = False
-    ORG_DEFAULT_SUBSCRIPTION_PLAN_SLUG = 'trial-v2-monthly'
+    RTD_ORG_DEFAULT_STRIPE_PRICE = 'trial-v2-monthly'
```

Comment:
the stripe id was the same as the slug :D

Reply:
yeah, this was done on purpose when we created these objects on Stripe